js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
+  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+
+  defaultStyle();
+
+  if (this.value === '') {
+    styleSeting('email', 'empty');
```

Comment:
`만약에 email값이 없으면, empty 스타일을 적용해줘` 라는 요구사항에서 여기 코드는 `empty 스타일을 적용해줘` 부분인 것 같은데요~ 아래와 같이 사용해주는건 어떨까요?

```suggestion
    setEmptyEmailStyle();
```